src/js/components/SelectMultiple/__tests__/__snapshots__/SelectMultiple-test.tsx.snap

```diff
@@ -1045,7 +1037,7 @@ exports[`SelectMultiple keyboard interactions 1`] = `
     <button
       aria-expanded="true"
       aria-haspopup="listbox"
-      aria-label="Open drop. 2 selected"
+      aria-label="Open drop. 0 selected"
```

Comment:
Looks like we may need to update the keyboard interactions test. It looks like it is trying to navigate into the list items by pressing down arrow from the search input, which we are no longer supporting

Reply:
Oops, thank you I will fix.

Reply:
Done.